packages/web/components/Dashboard/Post/PostComment.tsx

```diff
@@ -216,6 +217,37 @@ const PostComment: React.FC<PostCommentProps> = ({
           word-wrap: break-word;
         }
 
+        :global(.comment-body h1) {
+          font-size: 1.5em;
+          font-weight: bold;
```

Comment:
```suggestion
          font-weight: 600;
```

We generally prefer to specify font-weights, same thing on line 230